.github/workflows/ubuntu-bundler3.yml

```diff
@@ -0,0 +1,44 @@
+name: ubuntu-bundler3
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - staging
+      - trying
+
+jobs:
+  ubuntu-bundler3:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ 2.4.9, 2.5.7, 2.6.5 ]
+        rgv: [ v3.0.6, master ]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - name: Prepare dependencies
+        run: |
+          bin/rake override_version
+          bin/rake spec:parallel_deps
+      - name: Run Test
+        run: bin/parallel_rspec spec
+        env:
+          RGV: ${{ matrix.rgv }}
```

Comment:
I think most steps should use `RGV`, so we could move it to an environment variable at the "job" level, instead of passing it to every step?